docs/airflow/README.md

```diff
@@ -1,5 +1,209 @@
+<br />
+
+<p align="center">
+  <img src=".../_static/images/ingestion.png" />
```

Comment:
Is this path still correct?

Reply:
Nope. Thanks for reminding me! Just fixed